classes/02class/exercises/c02-network07/danielvca/README.md

```diff
@@ -0,0 +1,78 @@
+﻿
+# c02-network07
+
+## Commands Execution Output
+
+- Commands for creating the route tables:
+```
+aws ec2 create-route-table \
+--vpc-id vpc-0c2c93451cdd28192
+```
+
+- Commands for associating the route tables with subnets:
+```
+aws ec2 attach-internet-gateway \
```

Comment:
you are not associating the route table here but rather attaching the IGW. Wrong reference?